addon/components/mediasuite-check-boxes.js

```diff
@@ -1,29 +1,18 @@
 import Ember from 'ember'
-const { computed } = Ember
+const { computed, get, set } = Ember
 
 export default Ember.Component.extend({
   tagName: 'fieldset',
-  classNames: ['check-boxes'],
-  isChecked: function (option) {
-    let initialState = Ember.A(this.get('initialState'))
-    return (initialState.indexOf(option.value) !== -1)
+  classNames: ['check-boxes', 'group-element-wrap'],
+  selected: computed.filterBy('options', 'checked', true),
+  didReceiveAttrs () {
+    this._super(...arguments)
+    set(this, 'clonedOptions', JSON.parse(JSON.stringify(get(this, 'options'))))
```

Comment:
I'm curious about your decision to do this in a lifecycle hook instead of with a computed property